proto/Cargo.toml

```diff
@@ -27,6 +27,6 @@ protobuf = "2"
 lazy_static = { version = "1.3", optional = true }
 
 [build-dependencies]
-protobuf-build = { version = "0.11", default-features = false }
-grpcio-compiler = { path = "../compiler", version = "0.5.0", default-features = false }
+protobuf-build = { version = "=0.11.3", default-features = false }
+grpcio-compiler = { version = "=0.5.0", default-features = false }
```

Comment:
This is wrong. It prevents from updating and testing local compiler.

Reply:
protobuf-build use grpcio-compiler from crates.io and locked it. So you mean patch grpcio-compiler with local crate in root cargo.toml?

Reply:
It can be a solution.